### DISCUSSION TITLE
iOS: Improvement of navigation start and stop.

### DISCUSSION THREAD

**Discussion by cooperbang** [organicmaps/organicmaps#9141] [Ideas]
1. When you stop the navigation, the blue marked path should not disappear, but continue to be displayed. It should be possible to restart navigation from the current location without re-entering the destination.

2. The button to stop the navigation should not be hidden in the burger menu but should be directly visible.

**Comment by biodranik** [comment]
1. Why? What is your use-case?
2. Why?

**Reply by cooperbang** [reply]
1. This is the normal behavior in Google or Apple Maps. Sometimes you want to stop the navigation and just have the route displayed (without navigation instructions). If you want to start the navigation again, you have to enter the destination again in Organic Maps - not in Google or Apple.

2. Because it is easier to end the navigation this way. It's one click less.

**Comment by biodranik** [comment]
1. Google returns back to the build/preview route screen. Apple just shows the destination info opened. 
2. Stop button takes additional space, not every app shows it. And there is a reason for that: you don't need this button often. 

Both these cases/behaviors can be improved separately. There is an issue about new UX for navigation mode.
